Chapter 1 and 2 corrections (#99)
chhoumann/p8: report/src/problem_analysis/problems_statement.tex

\section{Problem statement}\label{sec:problem_statement}

> In Section \ref{sec:understanding_the_problem} it was established that the stakeholders had several issues regarding the management of meetings. 

Reviewer: ```suggestion
In Section \ref{sec:understanding_the_problem}, it was established that the stakeholders have several issues regarding the management of meetings. 
```

To resolve these issues, the stakeholders proposed a system defined by several user stories detailed in Appendix \ref{appendix:user_stories}. 
To examine whether such a system would solve the stakeholders' issues, a hifi prototype was developed using Figma\cite{Figma} by following methods from user-centred design.
The prototype was tested using an approach to summative usability testing defined by \citeauthor{lazar2005web}~\cite{lazar2005web}. 
The results of this test was that, while many issues regarding usability could be resolved by a system based on the prototype, some still remained. 

> One issue that remained was that meeting rooms might still be unused due to reliance on user interactions for ending the meeting early.

Reviewer: ```suggestion
The most significant problem was that meeting rooms might continue to show as occupied in the system, despite being vacant, due to the dependence on user interaction for concluding meetings ahead of time.
```

It was concluded that a solution to Mariendal IT's problems should provide functionality for minimizing reliance on user interaction.

> To alleviate such issues, a technology that could automatically identify when a room is unused could be deployed. 

Reviewer: ```suggestion
To alleviate such issues, a technology that can automatically identify when a room is unused could be deployed. 
```
